### ISSUE TITLE
Download all files of a packages at once

### ISSUE THREAD

**Issue by scheidan** [eawag-rdm/eawag-ckan#23]
Some packages contain several files (e.g. `4s-project-collection-of-questionnaires`).
It would be good to have the functionality to download all files at once, maybe as zip.

As use case I see packages containing data, code and a paper. If want to repeat all computations and recompile the paper, I'll need all files, while others may only want the raw data.

**Comment by hvwaldow** [comment]
Yes, that is on the "list". I still have to figure out how to make public the "list" of features that are shortlisted for implementation.

**Comment by hvwaldow** [comment]
Peter wants that too
